apps/student/src/main/java/com/instructure/student/activity/LoginActivity.kt

```
@@ -67,6 +74,24 @@ class LoginActivity : BaseLoginInitActivity() {
         return BuildConfig.IS_TESTING
     }
 
+    override fun onCreate(savedInstanceState: Bundle?) {
+        val startNewRelic =
+                RemoteConfigUtils.getString(RemoteConfigParam.USE_REMOTE_CONFIG)?.equals("true",ignoreCase = true) ?: false
```

Can the `RemoteConfigUtils.getBoolean()` method not be used here? Might be slightly cleaner.

---

I'm trying to do something safe here.  I *think* that RemoteConfigUtils.getBoolean() will throw if the string does not contain a boolean value, and I want to avoid any chance of that.  

(That begs the question, "Should we *ever* use anything besides getString()?"  Maybe not, unless you're super-confident that the Remote Config value will always be set to a valid value. 
 Or, I suppose that we could change getBoolean(), etc..., to swallow exceptions and return some nominal error value if the string format is not digestible.)

This should just get used once at startup, so the extra few cycles shouldn't matter.

---

Oh, good point. I'd imagine we want all those "helpers" to be safe to use for inappropriate data types. Otherwise, we could do these casts and checks in multiple places if a flag covers a large area.

Looks like the source for `String.toBoolean()` shouldn't ever result in a crash. It will check if string is null and then if it equalsIgnoreCase to "true".
But again, this is a really minor nitpick and isn't really necessary to change. Just developer preference :)

---

I'm going to create a ticket to make sure that all of those helper methods are non-throwing.  (I believe you that getBoolean is non-throwing, but I'll need to look into the other two.)  Then we cal feel a little better about using them.